common/protob/messages-tezos.proto

```diff
@@ -60,7 +61,7 @@ message TezosSignTx {
     /*
      * Tezos contract ID
      */
-    message TezosContractID {
+     message TezosContractID {
```

Comment:
We can remove `TezosContractID` completely if it's not used anymore at all.

Reply:
We still use `TezosContractID` in the `TezosTransactionOp` as the type of the destination field

Reply:
What's the difference between Implicit and Originated contract type?

Reply:
Originated accounts(KT1) are created by the origination operation from an implicit account. The originated account can contain MICHELSON code, they're essentially smart contracts. 

Implicit accounts(tz{1, 2, 3}) are the 'classic' accounts, that are holding tokens and they can not include code (contract).

The contract type here is referring to the account type.